src/components/Chat/BottomSheetGif.js

```diff
@@ -27,18 +28,10 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
   const [list, setList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
 
-  const handleChangeText = (text) => {
-    setSearchText(text);
-  };
-
-  const handleOnClearText = () => {
-    setSearchText('');
-  };
-
-  const onLoad = async () => {
+  const onLoad = async (search) => {
     setIsLoading(true);
     try {
-      const result = await getGifFeatured();
+      const result = await getGifFeatured(search);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onLoad` function now accepts a `search` parameter which is passed to the `getGifFeatured` function. This is a good change as it allows for more specific GIF fetching. However, there's no error handling in case the `getGifFeatured` function fails. It would be beneficial to add a catch block to handle any potential errors.
```diff
   const onLoad = async (search) => {
     setIsLoading(true);
     try {
       const result = await getGifFeatured(search);
+    } catch (error) {
+      console.error(error);
+      // Handle the error appropriately here
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
No need